src/index.js

```diff
@@ -10,10 +10,23 @@ const addEventsUrl = baseUrl + '/addEvents';
 const uploadLogsUrl = baseUrl + '/api/uploadLogs';
 
 let logGroupOptions = {};
-if (process.env['LOG_GROUP_OPTIONS']) logGroupOptions = JSON.parse(process.env['LOG_GROUP_OPTIONS']);
-let defaultParserName = process.env['PARSER_NAME'];
+let userLogFile = null;
+let userParserName = null;
+let userServerHost = null;
+if (process.env['LOG_GROUP_OPTIONS']) {
+    logGroupOptions = JSON.parse(process.env['LOG_GROUP_OPTIONS']);
+    let functionLogGroupName = process.env['AWS_LAMBDA_LOG_GROUP_NAME'].substr(1);
```

Comment:
One thing I missed here - it's possible for AWS_LAMBDA_LOG_GROUP_NAME to be null while LOG_GROUP_OPTIONS isn't, in which case we'll get an exception here - we should probably handle that case gracefully